libs/blocks/marketo/marketo.js

```diff
@@ -156,13 +119,16 @@ const init = (el) => {
         formWrapper.append(description);
       }
 
-      const marketoForm = createTag('form', { ID: `mktoForm_${formID}`, class: 'hide-errors' });
-      formWrapper.append(marketoForm);
-      fragment.append(error, formWrapper);
+      const marketoForm = createTag('form', { ID: `mktoForm_${formID}`, class: 'hide-errors my-form', style: 'opacity:0;visibility:hidden;' });
```

Comment:
What's `my-form`?

Reply:
Good catch! That was from testing, so I've removed it